sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerItemProperties.java

```diff
@@ -13,9 +13,9 @@
 /**
  * Properties of a container.
  */
-@JacksonXmlRootElement(localName = "BlobContainerProperties")
```

Comment:
This was released using the wrong XML property wasn't it?

Reply:
Yes, it was supposed to be "Properties"